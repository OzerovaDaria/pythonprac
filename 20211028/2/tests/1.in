print(list(slide('ABCDEFabcdef', 5)))
