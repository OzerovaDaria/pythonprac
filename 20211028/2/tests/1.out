['A', 'B', 'C', 'D', 'E', 'B', 'C', 'D', 'E', 'F', 'C', 'D', 'E', 'F', 'a', 'D', 'E', 'F', 'a', 'b', 'E', 'F', 'a', 'b', 'c', 'F', 'a', 'b', 'c', 'd']
